applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.cpp

```diff
@@ -567,15 +567,13 @@ void ConstitutiveLawUtilities<6>::CalculateProjectionOperator(
     indexes_ij(0, 1) = 1;
     indexes_ij(1, 0) = 1;
     indexes_ij(1, 1) = 2;
-    indexes_ij(2, 0) = 0;
-    indexes_ij(2, 1) = 2;
 
     IndexType i, j;
     double h_i = 0.0, h_j = 0.0;
     Matrix cross_p_ij_tensor;
     Vector cross_p_ij_vector;
 
-    for (IndexType index = 0; index < Dimension; ++index) {
+    for (IndexType index = 0; index < 2; ++index) {
```

Comment:
of maybe directly?
for (IndexType i=0; i< Dimension; ++i) {
for (IndexType j=i; j< Dimension; ++j) {

Reply:
wait this also does not work. It should be something like this. This Sum is very confusing...
for (IndexType i=0; i< Dimension - 1; ++i) {
for (IndexType j=i + 1; j< Dimension; ++j) {